docs/publish-module/publish-information/meta-data-information.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Meta Data Information
+description: Detailed guide on filling out meta data for app distribution platforms within the Appcircle dashboard.
+tags: [meta data, app distribution, appcircle dashboard]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Meta Data Information
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3667-meta-data-information-menu.png' />
+
+## Overview
+
+The Meta Data Information section in the Appcircle dashboard is essential for defining the presence of your application on digital distribution platforms. This document will guide you through each field and its significance to ensure your app's metadata is complete and effective.
+
+:::note
+The Meta Data Information section is available for iOS apps only currently. We will be adding support for Android apps in the future.
+:::
+
+## iOS Meta Data Information
+
+### Localization Settings
+
+The localization dropdown allows you to select the language in which you want to present your app’s metadata on the App Store. This feature supports multiple languages, ensuring that you can target specific demographics and cater to a global audience.
+
+When you select a language, you will provide localized versions of your app's metadata, including promotional texts, descriptions, and what’s new in this version. Localization helps in reaching a wider audience by providing information in the users' native language.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3667-meta-data-information-localization.png' />
+
+---
+
+### Fields Explained
+
+#### iOS Previews and Screenshots
+
+- **Drag and Drop**: Upload up to 3 app previews and 10 screenshots.
+  - **iPhone**: Supported resolutions are 1290 x 2796px or 2796 x 1290px.
+  - **iPad**: Supported resolutions are 2048 x 2732px or 2732 x 2048px.
+
+#### App Information
+
+- **Promotional Text**: Optional text to inform App Store visitors about any current app features without requiring an updated submission.
+- **Description**: Provide a detailed description of your app, highlighting its features and functionalities.
+- **What’s New**: Describe what’s new in this version of your app, such as new features, improvements, and bug fixes.
```

Comment:
```suggestion
- **What’s New**: Describe what’s new in this version of your app, such as new features, improvements, and bug fixes. 
::: info
Please note, this field is constantly visible on Appcircle. However, on App Store Connect, it will only appear when a new version is released. For this reason, if you are not going to release a new version, you may not see the information you enter here on App Store Connect.
:::
```